### PR TITLE
Code review trial

### PR DESCRIPTION
**Pull request by rmjspecial**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+          type="radio"
+          class="input-radio"/>Maybe</label>
+
+      <label><input
+          name="user-recommend"
+          value="not-sure"
+          type="radio"
+          class="input-radio"/>Not sure</label>  
+          
+             <p>
```
Comment (rmjspecial) [inline index.html:53]: line 53 I think your paragraph tag <p> is not properly arranged.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        <span class="clue">(Check all that apply)</span>
+      </p>
+
+      <label><input
+          name="prefer"
+          value="front-end-projects"
+          type="checkbox"
+          class="input-checkbox"/>Front-end Projects</label >
+    
+
```
Comment (rmjspecial) [inline index.html:64]: I think the spacing is too much pls do well and arrange properly thanks.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+          value="city-meetups"
+          type="checkbox"
+          class="input-checkbox"/>City Meetups</label >
+      <label><input
+          name="prefer"
+          value="wiki"
+          type="checkbox"
+          class="input-checkbox"/>Wiki</label>
+      
+
```
Comment (rmjspecial) [inline index.html:96]: I think your spacing is too much pls try and make the proper adjustment thanks.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    font-weight: bold;
+    color: orange;
+  }
+  .surv{
+    background: var(--color-darkblue-alpha);
+    padding: 2.5rem 0.625rem;
+    border-radius: 0.25rem;
+    text-align: center;
+  }
+
```
Comment (rmjspecial) [inline css/style.css:17]: I think you should try and arrange your code in a more readable way but in all, you did well thanks.

**Review by rmjspecial — COMMENTED** [review]
Changes Required♻
please check my comment below and make all required changes.

submit for re-review when you are done.

thanks😊

File: index.html (hunk 3 above)
Comment (m05e5) [reply index.html:96]: Hi Hi. Great job with your reviews they are kind and polite. I only suggest that when giving a review you always include an explanation, a "why", for each requested change.
Example: Nice work. I suggest removing unnecessary empty lines in your code to keep your code clean and professional.

File: index.html (hunk 1 above)
Comment (m05e5) [reply index.html:53]: Example: Hi. Please I suggest giving your code a proper indentation. By doing so, your code will be easier to understand for you and for your collaborators

**Review by m05e5 — APPROVED** [review]
# Hi @rmjspecial , 

### Status: Approved ✔️
goodjob



Your project is complete! 
Congratulations! 🎉 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] _(optional)_ Please Kindly give your PR a descriptive message. It makes it easier for other people to understand what you did

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**